openfl-tutorials/Federated_Keras_MNIST_Tutorial.ipynb

```diff
@@ -69,8 +94,12 @@
     "#Import training and validation images/labels\n",
     "train_images = mnist.train_images()\n",
     "train_labels = to_categorical(mnist.train_labels())\n",
-    "valid_images = mnist.test_images()\n",
-    "valid_labels = to_categorical(mnist.test_labels())\n",
+    "\n",
+    "valid_images = mnist.test_images()[0:8000,:,:]\n",
```

Comment:
I would also suggest adding a variable to handle 8000.

Reply:
@alexey-gruzdev  good catch! thanks for the suggestion.